src/compiler/sys.ts

```diff
@@ -1604,7 +1604,11 @@ export let sys: System = (() => {
             setTimeout,
             clearTimeout,
             clearScreen: () => {
-                process.stdout.write("\x1Bc");
+                process.stdout.write(
+                    platform === "win32"

```

Comment:
Like https://github.com/microsoft/TypeScript/pull/57701 this will be a behavior change; we'll have to check to see how it behaves and make sure we're okay with the change.